docs/ReleaseNotes.md

```diff
@@ -13,6 +13,7 @@ Bug Fixes:
 * Fixed the long name of the pounds per square inch (psi) of pressure measure to match the long name used
   in CLDR so that measures of psi can be formatted with the format templates from CLDR
 * Fixed the ilib demo build failure issue that is related ilib-scanner.
+* Update to time zone data 2022a
```

Comment:
Shouldn't this be in `New Feaatures` not Bug fix?

Reply:
It's not really a new feature. It is getting an existing feature to work properly again. (time zones). If you look at all the previous releases, the time zone updates are in the bug fix sections

Reply:
I see. When I checked the previous releasenotes, It was mixed that's why I asked it.